app/src/components/Tabs/Activity/activity-tab.tsx

```diff
@@ -77,8 +83,8 @@ const ActivityTab: FC<ActivityTabProps> = ({
 
   const { methodologies, directMeasure } = getMethodologies();
 
-  const getSuggestedActivities = () => {
-    if (!selectedMethodology) return [];
+  const getSuggestedActivities: () => SuggestedActivity[] = () => {
+    if (!selectedMethodology) return [] as SuggestedActivity[];
```

Comment:
```suggestion
  const getSuggestedActivities = (): SuggestedActivity[] => {
    if (!selectedMethodology) return [];
```

And you can remove the cast below as well if you define the return type of the lambda I think.